### PR TITLE
Update enable-your-device-for-development.md

### PR DESCRIPTION
**Pull request by bergano65**
more consistent emphasizing

### CODE REVIEW

**Comment by PRMerger16** [comment]
@bergano65 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows-apps-src/get-started/enable-your-device-for-development.md [hunk 1]
```diff
@@ -46,18 +46,18 @@ Here's the settings page on the desktop device family:
...
  You can enable a device for development, or just for sideloading.
 
--   *Microsoft Store apps* is the default setting. If you aren't developing apps, or using special internal apps issued by your company, keep this setting active.
--   *Sideloading* is installing and then running or testing an app that has not been certified by the Microsoft Store. For example, an app that is internal to your company only.
--   *Developer mode* lets you sideload apps, and also run apps from Visual Studio in debug mode.
-
+-   **Microsoft Store apps** is the default setting. If you aren't developing apps, or using special internal apps issued by your company, keep this setting active.
+-   **Sideloading** is installing and then running or testing an app that has not been certified by the Microsoft Store. For example, an app that is internal to your company only.
+-   **Developer mode** lets you sideload apps, and also run apps from Visual Studio in debug mode.
+1
```
Comment (bergano65) [inline windows-apps-src/get-started/enable-your-device-for-development.md:52]: sorry, my mistake

**Comment by ktoliver** [comment]
@QuinnRadich - Could you review the proposed changes? (Note that at least four typos in the updates introduce errors, at lines 52, 135 (branding}, 138 (breaks a link), and 204.) Thanks.

File: windows-apps-src/get-started/enable-your-device-for-development.md [hunk 2]
```diff
@@ -118,27 +118,27 @@ The existing SSH server used in Windows is not yet protocol compliant, so using
...
 
 
 ## Notes
-In early versions of Windows 10 Mobile, a Crash Dumps option was present in the Developer Settings menu.  This has been moved to [Device Portal](../debug-test-perf/device-portal.md) so that it can be used remotely rather than just over USB.  
+In early versions of Windows 10 Mobile, a **Crash Dumps** option was present in the **Developer Settings** menu.  This has been moved to [Device Portal](../debug-test-perf/device-portal.md) so that it can be used remotely rather than just over USB.  
 
-There are several tools you can use to deploy an app from a Windows 10 PC to a Windows 10 device. Both devices must be connected to the same subnet of the network by a wired or wireless connection, or they must be connected by USB. Both of the ways listed install only the app package (.appx/.appxbundle); they do not install certificates.
+There are several tools you can use to deploy an app from a Windtows 10 PC to a Windows 10 device. Both devices must be connected to the same subnet of the network by a wired or wireless connection, or they must be connected by USB. Both of the ways listed install only the app package (.appx/.appxbundle); they do not install certificates.
```
Comment (bergano65) [inline windows-apps-src/get-started/enable-your-device-for-development.md:135]: sorry my mistake

File: windows-apps-src/get-started/enable-your-device-for-development.md [hunk 3]
```diff
@@ -118,27 +118,27 @@ The existing SSH server used in Windows is not yet protocol compliant, so using
...
 
 -   Use the Windows 10 Application Deployment (WinAppDeployCmd) tool. Learn more about [the WinAppDeployCmd tool](https://docs.microsoft.com/previous-versions/windows/apps/mt203806(v=vs.140)).
--   You can use [Device Portal](../debug-test-perf/device-portal.md) to deploy from your browser to a mobile device running Windows 10, Version 1511 or later. Use the **[Apps](../debug-test-perf/device-portal.md#apps-manager)** page in Device Portal to upload an app package (.appx) and install it on the device.
+-   You can use [Device Portal](../debug-test-perf/device-portal.md) to deploy from your browser to a mobile device running Windows 10, Version 1511 or later. Use the **[Apps]1(../debug-test-perf/device-portal.md#apps-manager)** page in Device Portal to upload an app package (.appx) and install it on the device.
```
Comment (bergano65) [inline windows-apps-src/get-started/enable-your-device-for-development.md:138]: sorry, mistake

File: windows-apps-src/get-started/enable-your-device-for-development.md [hunk 4]
```diff
@@ -202,7 +201,7 @@ You can use gpedit.msc to set the group policies to enable your device, unless y
 1.  Run **regedit**.
 2.  To enable sideloading, set the value of this DWORD to 1:
 
-    -   `HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock\\AllowAllTrustedApps`
+    -   `HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock\\AllowAllTrustedA pps`
```
Comment (bergano65) [inline windows-apps-src/get-started/enable-your-device-for-development.md:204]: sorry, mistake

Comment (bergano65) [reply windows-apps-src/get-started/enable-your-device-for-development.md:204]: fixed, but suggested more consistent emphasizing seems more appropriate

**Comment by QuinnRadich** [comment]
There's some good changes in this PR, and some things that need further work / don't match our style guidelines. It's convinced me that I need to make a pass on this article to make sure the emphasis conforms to our doc standards, but I think it's more efficient if I handle that on my own rather than going back and forth trying to sort it out with you.

Thank you.

**Pull request closed without merging**